src/current/releases/index.md

```diff
@@ -112,10 +112,10 @@ As of 2024, CockroachDB is released under a staged delivery process. New release
 
 | Version | Release Type | GA date | Latest patch release |
 | :---: | :---: | :---: | :---: |
-| [v24.2](#v24-2) | Innovation | 2024-08-12 | v24.2.0 |
-| [v24.1](#v24-1) | Regular | 2024-05-20 | v24.1.4 |
-| [v23.2](#v23-2) | Regular | 2024-02-05 | v23.2.10 (LTS) |
-| [v23.1](#v23-1) | Regular | 2023-05-15 | v23.1.25 (LTS) |
+| [v24.2](#v24-2) | Innovation | 2024-09-25 | v24.2.3 |
```

Comment:
These dates do not look right to me when I look at the `versions.csv`, where the `release_date` for v24.2 is listed as `2024-08-12`, for v24.1 it is listed as `2024-05-20`, for v23.2 it is listed as `2024-02-05`, etc. Where are you seeing these other dates? 

The `Latest patch release` column is not ideal as it needs to be manually maintained. Is the column valuable enough to take that on, or to take on cycles to automate it?

Reply:
You're right; I updated these to the latest patch release date in error!

Also, you have a good point about maintainability — I think we should drop the column and I'll create an issue to consider automated display of it, instead.

I'll close this PR and open another, if that sounds good.

Reply:
SGTM